githubteacher-orlando.md

```diff
@@ -0,0 +1,18 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy
+- 4Rivers
+
+### Tourist Attractions
+
+- Disney World
+- Universal Studios Orlando
+- Legoland
+
+### Local Attractions
+
+- Gatorland
+- Payne's Praire in Gainesville
```

Comment:
@githubteacher Gainesville is pretty far away -- maybe you shouldn't include it? But it's not a blocker.